### PR TITLE
Feature/Textarea Simple component DAO-547

### PR DESCRIPTION
**Pull request by RakeshUP**

### CODE REVIEW

File: packages/ui-components/__tests__/textarea.test.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+
+import {TextareaSimple} from '../src';
+
+describe('TextareaSimple', () => {
+  function setup(args: any) {
+    render(<TextareaSimple {...args} />);
+    return screen.getByRole('textbox');
```
Comment (Fabricevladimir) [inline packages/ui-components/__tests__/textarea.test.tsx:9]: Whoa I wasn't aware of this selector! Nice

File: packages/ui-components/src/components/textarea/textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export type TextareaSimpleProps = {
+};
+
+export const TextareaSimple = styled.textarea.attrs({
+  className: `py-1.5 px-2 rounded-xl resize-none w-full border-2 border-ui-100 hover:border-ui-300 
+    disabled:bg-ui-100 disabled:border-ui-200 focus:border-primary-500 bg-white text-ui-600`,
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/textarea/textarea.tsx:8]: Missed the active state and the ever-present focus ring

**Review by Fabricevladimir — APPROVED** [review]
Nice work! Good to merge once the focus ring is added.

File: packages/ui-components/src/components/textarea/textarea.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  style: {minHeight: 144},
```
Comment (sepehr2github) [inline packages/ui-components/src/components/textarea/textarea.tsx]: You could set this style in the backtick section :D.

**Review by sepehr2github — APPROVED** [review]
Hooray! The new textarea components are ready! Thank you 👍

File: packages/ui-components/src/components/textarea/textarea.tsx (hunk 2 above)
Comment (RakeshUP) [reply packages/ui-components/src/components/textarea/textarea.tsx:8]: So I saw the designs and there are two variants, focus and active for textarea. But if we specify focus styles, active is not applied at all. And active according to the designs looks like when the value is being entered in the input. I've put the active spec in design in the focus variant in code. 

I'll have to talk to Selim and see what to do about this

**Pull request merged**